openslides/core/static/js/core/pdf.js

```diff
@@ -185,12 +192,26 @@ angular.module('OpenSlidesApp.core.pdf', [])
             // Used placeholder for currentPage and pageCount which
             // are replaced by dynamic footer function in pdf-worker.js.
             var getFooter = function() {
+                var columns = [];
+
+                // TODO: This does't work. The image does not get rendered.
+                if (logoFooterUrl) {
```

Comment:
@tsiegleauq @emanuelschuetze Can you have a look how to insert the footer image correctly into the document? You may also change the `pdf-worker.js`; see my changes there.